style.css

```diff
@@ -1 +1,116 @@
-/*  Add your styling here */
+body {
```

Comment:
Hi Farnoosh, good try on this PR, but it seems like your page is not responsive, can you work on that?

Reply:
I have worded on that, I look forward to hear your suggestions:)